x/gov/keeper.go

```
@@ -120,32 +118,61 @@ func (keeper Keeper) NewTextProposal(ctx sdk.Context, title string, description
 	keeper.InsertInactiveProposalQueue(ctx, proposal.GetDepositEndTime(), proposalID)
 	return proposal
 }
+*/
+
+func (keeper Keeper) submitProposal(ctx sdk.Context, content ProposalContent) (proposal Proposal, err sdk.Error) {
+	proposalID, err := keeper.getNewProposalID(ctx)
+	if err != nil {
+		// XXX: should we panic here?
```

No, we should return an error. Let's remove this.